services/src/main/scala/cromwell/pipeline/service/ControllerClient.scala

```diff
@@ -0,0 +1,84 @@
+package cromwell.pipeline.service
```

Comment:
Why did you choose the service module for this?

Reply:
Trait was here and I consulted with Masha about it